moduleroot/.gitattributes

```diff
@@ -0,0 +1,5 @@
+#This file is generated by ModuleSync, do not edit.
+*.rb  eol=lf
+*.erb eol=lf
+*.pp  eol=lf
+*.sh  eol=lf
```

Comment:
Looks like this is causing issues with one of our modules. Looks like we'll need to fix that one up.